dvc/command/check_ignore.py

```diff
@@ -61,9 +106,24 @@ def add_parser(subparsers, parent_parser):
         help="Show the target paths which don’t match any pattern. "
         "Only usable when `--details` is also employed",
     )
+    parser.add_argument(
+        "--stdin",
+        action="store_true",
+        default=False,
+        help="Read pathnames from the standard input, one per line, "
+        "instead of from the command-line.",
+    )
```

Comment:
What would be the typical use cases for --stdin? I've honestly never used it in git's check-ignore, so I'm not sure I understand who and why would use it. Just worried about a potentially unused/dead feature. `--all` is very useful, no doubt.

Reply:
@efiop 
Most use cases are inside a Linux pipeline, I think.
```
find . -ctime -7 | dvc check-ignore --stdin 
```